src/Swipeable.js

```diff
@@ -48,9 +48,9 @@ export default class Swipeable extends PureComponent {
 
   handleTouchEnd(e) {
     const {
-      minDistance = 20,
```

Comment:
@ankitsilaich thanks for the contribution. Your suggestion of using `defaultProps` is logical.
I have intentionally left the defaults inlined like that here, though. 

**My motivation is the following:**
 - I would use `defaultProps` only if the prop is used in more than one place (method)
 - inlining them like that makes it easy to digest at once since everything is contained in a single method
 - `handleTouchEnd` is where all the interesting things happen, so I wanted useful information to be there.

What do you think?

Reply:
@moubi Thanks for putting suggestions on the PR. I got your concern. What about keeping empty functions in default props to avoid check like ```js  onSwipeLeft && onSwipeLeft()``` and moving timeout default back to handleTouchEnd

Reply:
I was 50:50 about leaving defaults for the functions. Performance-wise there is very small difference. If we introduce the defaults it means a function call for each swipe no matter if consumer component has subscribed for that event. I would like us to avoid that. 
We may also need to introduce the babel plugin as well.

Let's keep it as it is for now.  

And thank you for using the issues. I think this is the right way to communicate possible changes/feature requests.

What do you think about replacing `minDistance/maxDistance` with `minXDistance, maxXDistance, minYDistance, maxYDistance`? So you see any real use case example?

Reply:
@moubi Thanks for the detailed feedback. Talking about ```minDistance/maxDistance``` , I think for now it solves the purpose. I am not sure if there any use case where we want to configure different values for these. So you can keep minDistance/maxDistance for now. Thanks again for giving your feedback on the PR.